### PR TITLE
Add fields to explicitly specify username and email for commit user

### PR DESCRIPTION
**Pull request by ahmedwaleedmalik**

### CODE REVIEW

**Comment by ahmedwaleedmalik** [comment]
@stefanprodan This would be a nice addition since for automated workflows/actions user can prefer to use a bot/static account for cross-repo commits i.e. commits to the charts repository.

File: action.yml [hunk 1]
```diff
@@ -32,6 +32,14 @@ inputs:
   linting:
     description: "Toggle Helm linting, can be disabled by setting it to 'off'"
     required: false
+  commit_username:
+    description: "The user name used for the commit user"
+    required: false
+    default: "$GITHUB_ACTOR"
```
Comment (stefanprodan) [inline action.yml:38]: Does GH Actions support env vars here?

Comment (ahmedwaleedmalik) [reply action.yml:38]: No it doesn't, just wanted to represent what the default value would be.

Comment (ahmedwaleedmalik) [reply action.yml:38]: Nevermind i think it does, let me test it out.

Comment (ahmedwaleedmalik) [reply action.yml]: It does, updated the PR as discussed

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @ahmedwaleedmalik

**Pull request merged**